corehq/warehouse/transforms/sql/location_dim.sql

```diff
@@ -0,0 +1,97 @@
+-- First insert all records into location dim from staging tables without foreign keys
+INSERT INTO {{ location_dim }} (
```

Comment:
does upsert require upgrade? i think we can also get around using upsert (and doing an upgrade everywhere) by just using some combination of `update from...` and `delete from using...` queries